File: arrayMaxConsecutiveSum/arrayMaxConsecutiveSum.js
function arrayMaxConsecutiveSum(arr) {
    let max = 0;
    for (let i = 0; i < arr.length - 1; i++) {
        if ((arr[i] + arr[i + 1]) > max) {
            max = arr[i] + arr[i + 1];
        }
    }
    return max;
}
console.log(arrayMaxConsecutiveSum([1, 2, 3, 4, 5]));

File: arrayMaxConsecutiveSum/arrayMaxConsecutiveSum.py
def arrayMaxConsecutiveSum(arr):
    maxEl = 0
    for i in range(0,len(arr)-1):
        if(arr[i] + arr[i+1] > maxEl):
            maxEl = arr[i] + arr[i+1]
    return maxEl

print(arrayMaxConsecutiveSum([1, 2, 3, 4, 5]));